Sources/Commands/UserToolchain.swift

```diff
@@ -34,10 +35,16 @@ struct UserToolchain: Toolchain {
 
     init() throws {
         do {
+
             SWIFT_EXEC = getenv("SWIFT_EXEC")
                 // use the swiftc installed alongside ourselves
                 ?? Path.join(Process.arguments[0], "../swiftc").abspath
 
+            lldb = getenv("LLDB")
+                // use the lldb installed alongside ourselves
+                ?? Path.join(Process.arguments[0], "../lldb").abspath
```

Comment:
I would rather call `swift` to launch the REPL than depend on how lldb is launched.